src/components/button/Button.css

```diff
@@ -0,0 +1,42 @@
+.button1 {
+    font-size: 16px;
+    background: #171717;
+    border: 3px solid #171717;
+    color: #FFFFFF;
+    border-radius: 7px;
+    padding: 12px 27px;
+    box-sizing: border-box;
+    width: max-content;
+    min-width: 150px;
+    display: flex;
+    justify-content: flex-start;
```

Comment:
Dit is de default waarde van `justify-content`, dus kun je weglaten :)

Reply:
Weggehaald